### ISSUE TITLE
Update homebrew formula to install the influx-cli tool

### ISSUE THREAD

**Issue by russorat** [influxdata/influx-cli#211]
we should update the homebrew formula https://github.com/Homebrew/homebrew-core/blob/master/Formula/influxdb.rb with the ability to install the cli from this repo.

we should also consider a separate influxdb-cli only brew command, or a way to just install the cli without the oss influxdb binary.

**Comment by danxmoran** [comment]
@russorat I think a separate formula is probably the way to go. My thinking is we'd have a new CLI formula, and update the existing `influxdb` formula to mark the CLI as a dependency. Some details/questions:
* Formulas hosted in `homebrew-core` aren't allowed to declare optional dependencies (see the `NOTE:` at the bottom of the docs [here](https://docs.brew.sh/Formula-Cookbook#specifying-other-formulae-as-dependencies)), so `influxdb` would need to declare a required dependency on the new CLI. That seems fine to me, what do you think?
* What should the new formula be called? I was going to use `influx-cli` to match the GitHub repository's name, but `influxdb-cli` ties more closely to the existing product/formula.

**Comment by russorat** [comment]
re: required dependency, seems fine to me

re: naming
it's unfortunate that the repo, the command, and the formula don't match. 

using `brew search` with `influx` or `influxdb` 
```
➜  openapi git:(master) ✗ brew search influx
==> Formulae
influxdb ✔                                                     influxdb@1                                                     flux
➜  openapi git:(master) ✗ brew search influxdb
==> Formulae
influxdb ✔                                                                                    influxdb@1
```

The only comparison i can think of is `kubernetes-cli` which is the `kubectl` command and comes from the kubernetes repo.

but running `brew install kubectl` and `brew install kubernetes-cli` are equivalent and i'm not sure how they did that.

i think i would call the formula `influxdb-cli` to keep it consistent and maybe try to figure out how to alias it similar to kubectl and kubernetes-cli.

**Comment by russorat** [comment]
ah here we go. we can manually create aliases for influx and influxd https://github.com/Homebrew/brew/issues/469#issuecomment-231151368

**Comment by danxmoran** [comment]
Closed by Homebrew/homebrew-core#82141
